### DISCUSSION TITLE
Route cache spliting to mobile / desktop / etc

### DISCUSSION THREAD

**Discussion by odranoelBR** [nitrojs/nitro#2432] [General]
I have my SWR running perfect and saving all the HTML on redis. 
But now i want split the HTML in two versions with v-if="deviceSize" 
![image](https://github.com/unjs/nitro/assets/7774534/8f1c63e7-6627-4500-91f2-f0bdd82e78de)

So i can have a more specific components and better performace because they have less html, less js, less css.
Is there a way ?

**Comment by pi0** [comment]
Hi! I think it would be best if you ask this in Nuxt discussions or discord channel 🙏🏼

**Comment by odranoelBR** [comment]
Hi @pi0, 
I asked here because the route cache mechanism is on nitro, right ? 
https://nitro.unjs.io/guide/cache#caching-route-rules

By the way i found a related question https://github.com/unjs/nitro/discussions/1170
